### ISSUE TITLE
Multi Line imports with unused modules are not detected as unused when they are on the second line

### ISSUE THREAD

**Issue by jmfederico** [SublimeLinter/SublimeLinter-pyflakes#2]
Hello

There is a problem with Pyflakes linter where multiline imports are not marked as unused when unused import is not on the first line. Example:

In this example only "UnusedOne" is marked as unused, but UnusedTwo is not:

```
from almacen.models import UsedOne, UnusedOne,\
                            UsedTwo, UnusedTwo
```

This is because UnusedTwo is reported by pyflakes as being on the previous line, which is never matched by the regex.

**Comment by braver** [comment]
Cannot reproduce on current versions of pyflakes, SL, and this plugin.

<img width="610" alt="screen shot 2018-05-17 at 15 29 34" src="https://user-images.githubusercontent.com/2543659/40180441-206a9f32-59e7-11e8-8c1b-812e0ed7aa02.png">
